awx/api/serializers.py

```diff
@@ -4888,6 +4888,14 @@ class InstanceGroupSerializer(BaseSerializer):
         read_only=True
     )
     instances = serializers.SerializerMethodField()
+    is_controller = serializers.BooleanField(
+        help_text=_('Indicates whether instance group controls any other group'),
+        read_only=True
+    )
+    is_isolated = serializers.BooleanField(
+        help_text=_('Indicates whether all instance nodes of the group are isolated'),
```

Comment:
I would say

> Indicates whether instances in this group are isolated. Isolated groups have a designated controller group.

Reply:
^ good call. Isolated groups are an all or nothing thing. Either they're all isolated or all regular instances, you can't have a mixed group